IFC4x3/Properties/h/HasConnectorA_2Bkoo$GzT9FxN8$7msTZGo/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HasConnectorA_2BkooGzT9FxN87msTZGo" Name="HasConnectorA" UniqueId="8bbb2cbf-43d7-493f-b5c8-fc7c36763432" PrimaryDataType="IfcBoolean" />
```

Comment:
if you use a connector pset the existance of a connector property would provide this detail so this is redundent